app/lib/audit/catalog_to_moab.rb

```diff
@@ -45,10 +44,12 @@ def check_catalog_version
       compare_version_and_take_action
     end
 
-    alias storage_location storage_dir
-
     private
 
+    def storage_location
+      complete_moab.moab_storage_root.storage_location
+    end
```

Comment:
hmm, so actually, nothing even appears to call `storage_location` anymore.  i think we could get rid of the method entirely.  nothing in this class seems to refer to it, afaict, and grepping through the codebase, only one thing seems to instantiate this class:
https://github.com/sul-dlss/preservation_catalog/blob/5398420867e2c91f4dc594643dfc698500bf9fbd/app/jobs/catalog_to_moab_job.rb#L14-L16

and nothing there references `storage_location` .